src/signals/incident/components/form/MapSelectors/Asset/Selector/Selector.tsx

```diff
@@ -131,14 +135,17 @@ const Selector = () => {
   const [showSelectionPanel, setShowSelectionPanel] = useState(true)
   const [pinMarker, setPinMarker] = useState<MarkerType>()
   const [map, setMap] = useState<MapType>()
+  const addressValue = address ? formatAddress(address) : ''
+
   const showMarker =
-    coordinates && (!selection || selection.type === UNREGISTERED_TYPE)
+    coordinates !== undefined &&
+    (!selection || selection.type === UNREGISTERED_TYPE)
```

Comment:
is the `!== undefined` really necessary here?

Reply:
Not necessarily, no.